backend/restAPI/restApi.go

```diff
@@ -155,8 +155,8 @@ func closeDbConnection(database *sql.DB) {
 	}
 }
 
-func (restApi *restAPIImpl) openDBConnection(c *gin.Context) (*sql.DB, error) {
-	database, err := sql.Open("exasol", exasol.NewConfig(c.GetString("dbUser"), c.GetString("dbPass")).Port(c.GetInt("dbPort")).Host(c.GetString("dbHost")).String())
+func (restApi *restAPIImpl) openDBConnection(c *gin.Context, autocommit bool) (*sql.DB, error) {
```

Comment:
Would it make sense to always turn off autocommit? So we don't need the parameter.
This would ensure that extensions are not installed half if something goes wrong.
Will require and explicit commit.